README.md

```diff
@@ -29,9 +29,11 @@ $ bundle exec jekyll serve
 
 Commits and merges into `master` will be deployed to the production server, current process is:
 
-1. Remove existing `_site` and archived versions `100days.tar.gz` in folder (both of these should not be under version control)
+### Manual
+
+1. Remove existing archive file `100days.tar.gz` in folder (this should not be under version control)
     ```bash
-    $ rm -rf _site/ 100days.tar.gz
```

Comment:
It might be nice to indicate the 'why' of when you might want to delete `_site/` first. It seems like if you forget to remove tar.gz before building then it is copied in again (and sometimes seems like it persists between builds?)

Reply:
Ah, good call! Added the tarball to `site.exclude`. (I've confirmed that the code of the `jekyll build` command blows away everything except files specified in `site.keep_files`.)